data/txHandler.go

```diff
@@ -203,6 +204,8 @@ func MakeTxHandler(opts TxHandlerOpts) (*TxHandler, error) {
 				uint64(opts.Config.TxBacklogAppTxPerSecondRate),
 				time.Duration(opts.Config.TxBacklogServiceRateWindowSeconds)*time.Second,
 			)
+			// set appLimiter triggering threshold at 50% of the base backlog size
+			handler.appLimiterBacklogThreshold = int(float64(opts.Config.TxBacklogSize) * float64(opts.Config.TxBacklogRateLimitingCongestionPct) / 100)
```

Comment:
One of my concerns here is that performance testing of application transactions will have to get a lot more sophisticated.

Reply:
I've run stateful_teal_medium test with 4 apps per node and 6k+ target appcall TPS, and got no TX messages drops because nodes managed to handle the load. In other words, if a node handles the load without backing up txpool and it's backlog, then no ARL in a play.